skyportal/handlers/api/internal/starlist_info.py

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
The pattern we've used so far is that each of the files in `skyportal/handlers` (& sub-directories) should contain request handler definitions, so I'd suggest removing this file, and either declaring this as a global in `source.py` (or `skyportal/offset_util.py` -- see my comment below on `util.py`), or parsing it from config.yaml. The handlers defined in this "internal" directory correspond to the handlers for the `/api/internal/*` endpoints.